frontend/packages/client/src/components/Community/StrategySelectorForm.js

```diff
@@ -1,24 +1,35 @@
 import React, { useEffect, useState } from 'react';
-import { useVotingStrategies } from 'hooks';
+import isEqual from 'lodash/isEqual';
 import { AddButton } from 'components';
 import { useModalContext } from 'contexts/NotificationModal';
 import StrategyEditorModal from './StrategyEditorModal';
-import StrategySelectorInput from './StrategySelectorInput';
+import StrategySelectorInput from 'components/Community/StrategySelectorInput';
+import { useVotingStrategies } from 'hooks';
 import { kebabToString } from 'utils';
 
 export default function StrategySelectorForm({
   existingStrategies = [],
   disableAddButton = false,
+  // this fc returns a component(Button) to render
   callToAction = () => {},
+  // callback to return strategies selected
+  onStrategySelection,
 } = {}) {
   // holds array of objects with strategy information
-  const [strategies, setStrategies] = useState(existingStrategies);
+  const [strategies, setStrategies] = useState([]);
 
-  // reloads lists if update is done
-  useEffect(() => {
+  // update state if new strategies are passed
+  useState(() => {
```

Comment:
I think this still be `useEffect`.

Reply:
good catch! i've removed the useEffect since props are not changing while component is loaded